templates/login.tpl

```diff
@@ -32,6 +32,14 @@
 						</div>
 					</div>
 				</div>
+				{{{each loginFormEntry}}}
+                <div class="form-group loginFormEntry">
+                    <label for="login-{loginFormEntry.styleName}" class="col-lg-4 control-label">{loginFormEntry.label}</label>
+                    <div id="login-{loginFormEntry.styleName}" class="col-lg-8">
+                        {{loginFormEntry.html}}
+                    </div>
+                </div>
+                {{{end}}}
```

Comment:
Tabs vs spaces?

Reply:
there, should match the file's indentation now